prevent URLs in attributes being escaped (#9820)
withastro/astro: .changeset/quick-islands-ring.md

---
"astro": patch
---

> Prevents URLs in attributes from being escaped in static builds

Reviewer: ```suggestion
Prevents fully formed URLs in attributes from being escaped
```

> Prevent URLs in `content` attributes being escaped for multiple params to work

Reviewer 2: ```suggestion
Prevents URLs in attributes from being escaped in static builds
```
I think it concerns any attribute, eg. hrefs

Author: Yeah I think you're right. I've just set it as a general URL check for any attribute for now.
